### PR TITLE
Client CHANGELOG v 0.4.0

### PR DESCRIPTION
**Pull request by NikolasKomonen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [0.4.0](https://github.com/redhat-developer/vscode-xml/milestone/4?closed=1) (February XX, 2019)
+
+### Enhancements
+
+* Modified schema validation messages. See [#91](https://github.com/redhat-developer/vscode-xml/issues/91).
+* Preference `xml.format.quotations` to set single vs double quotes for attribute values on format. See [lsp4xml#263](https://github.com/angelozerr/lsp4xml/issues/263).
+
```
Comment (fbricon) [inline CHANGELOG.md]: add xml.format.preserveEmptyContent

File: README.md [hunk 2]
```diff
@@ -59,13 +59,13 @@ The following settings are supported:
...
 * `xml.server.vmargs`: Extra VM arguments used to launch the XML Language Server. Requires VS Code restart.  
-  
-Since 0.3.0:
 * `xml.validation.enabled`: Set to `false` to disable all validation. Defaults to `true`.
 * `xml.validation.schema`: Set to `false` to disable schema validation. Defaults to `true`.
 * `xml.validation.noGrammar`: The message severity when a document has no associated grammar. Defaults to `hint`.
 * `xml.format.spaceBeforeEmptyCloseTag`: Set to `true` to insert space before the end of a self closing tag. Defaults to `true`.
 
+Since 0.4.0:
+* `xml.format.quotations`: Set to `doubleQuotes` to format and only use `"`. Defaults to `doubleQuotes`.
```
Comment (fbricon) [inline README.md]: add xml.format.preserveEmptyContent

remove xml.format.joinContentLines

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [0.4.0](https://github.com/redhat-developer/vscode-xml/milestone/4?closed=1) (February XX, 2019)
```
Comment (fbricon) [inline CHANGELOG.md]: add today's date

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [0.4.0](https://github.com/redhat-developer/vscode-xml/milestone/4?closed=1) (March 05, 2019)
```
Comment (fbricon) [inline CHANGELOG.md]: March 06 now

File: README.md [hunk 5]
```diff
@@ -53,19 +53,19 @@ The following settings are supported:
...
```
Comment (fbricon) [inline README.md]: Set to `doubleQuotes` to format and only use `"`, or `singleQuotes`  to format and only use `'`

**Comment by NikolasKomonen** [comment]
@fbricon updated

File: README.md [hunk 6]
```diff
@@ -53,19 +53,19 @@ The following settings are supported:
...
+* `xml.format.preserveEmptyContent`: Set to `doubleQuotes` to format and only use `"`, or `singleQuotes` to format and only use `'`. Defaults to doubleQuotes.
```
Comment (fbricon) [inline README.md]: NOOOPE

Comment (NikolasKomonen) [reply README.md]: @fbricon fixed

File: README.md [hunk 7]
```diff
@@ -53,19 +53,19 @@ The following settings are supported:
...
 * `xml.server.vmargs`: Extra VM arguments used to launch the XML Language Server. Requires VS Code restart.  
-  
-Since 0.3.0:
 * `xml.validation.enabled`: Set to `false` to disable all validation. Defaults to `true`.
 * `xml.validation.schema`: Set to `false` to disable schema validation. Defaults to `true`.
 * `xml.validation.noGrammar`: The message severity when a document has no associated grammar. Defaults to `hint`.
 * `xml.format.spaceBeforeEmptyCloseTag`: Set to `true` to insert space before the end of a self closing tag. Defaults to `true`.
 
+Since 0.4.0:
+* `xml.format.quotations`: Set to `doubleQuotes` to format and only use `"`, or `singleQuotes` to format and only use `'`. Defaults to doubleQuotes.
```
Comment (fbricon) [inline README.md]: ...Defaults to `doubleQuotes`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [0.4.0](https://github.com/redhat-developer/vscode-xml/milestone/4?closed=1) (March 06, 2019)
+
+### Enhancements
+
+* Modified schema validation messages. See [#91](https://github.com/redhat-developer/vscode-xml/issues/91).
+* Preference `xml.format.quotations` to set single vs double quotes for attribute values on format. See [lsp4xml#263](https://github.com/angelozerr/lsp4xml/issues/263).
+* Preference `xml.format.preserveEmptyContent` to preserve a whitespace value in an element's content. See [#96]([https://github.com/angelozerr/lsp4xml/issues/273](https://github.com/redhat-developer/vscode-xml/issues/96)).
+* Removed support for preference `xml.format.joinContentLines`.
```
Comment (fbricon) [inline CHANGELOG.md]: add https://github.com/angelozerr/lsp4xml/issues/288

**Pull request merged**